apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -45,6 +45,11 @@ type ClusterLogForwarderSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Forwarder Outputs",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:forwarderOutputs"}
 	Outputs []OutputSpec `json:"outputs,omitempty"`
 
+	// Filters are applied to log records passing through a pipeline.
+	// There are different types of filter that can select and modify log records in different ways.
+	// See [FilterTypeSpec] for a list of filter types.
+	Filters []FilterSpec `json:"filters,omitempty"`
```

Comment:
Maybe mention they are applied in order as defined

Reply:
Done.

Reply:
I don't see mention of order